Command/OPCacheResetCommand.php

```diff
@@ -0,0 +1,59 @@
+<?php declare(strict_types=1);
+
+namespace Leadin\SurvivalKitBundle\Command;
+
+use Leadin\SurvivalKitBundle\Logging\LogContext;
+use Leadin\SurvivalKitBundle\Logging\Logger;
+use Leadin\SurvivalKitBundle\HttpHelper\HttpClientHelper\HttpClient;
+use Leadin\SurvivalKitBundle\HttpHelper\HttpClientHelper\HttpClientException;
+
+use GuzzleHttp\RequestOptions;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+class OPCacheResetCommand extends Command
+{
+    protected static $defaultName = 'ssk:opcache-reset';
```

Comment:
no documentation about how to use, how to call it and why ?

should not delete 
https://github.com/proleadin/lp3/blob/8587ab060d593a192ac4a725c256c1f0b04af111/code/src/Controller/Api/InternalToolController.php

and 

https://github.com/proleadin/lp3/blob/60a21ce2028b359e0ab9c1b7e3234c95671197fd/code/src/Command/OPCacheResetCommand.php